.github/BOTMETA.yml

```diff
@@ -774,10 +1096,12 @@ files:
     maintainers: rdvencioneck $team_extreme
   $module_utils/network/f5:
```

Comment:
@acozine @abadger There are actually many files under `lib/ansible/module_utils/network/f5`, do you think we need to list them all, or is ijust the directory OK?

We only auto-generate docs for modules & plugins, so we might be OK just listering the directory

https://github.com/ansible/ansible/tree/devel/lib/ansible/module_utils/network/f5

Reply:
gotcha, they did takeover all of them and axed 2.

Reply:
Are all the module_utils files they migrated listed here?

Reply:
yes. however, 2 were removed. https://github.com/F5Networks/f5-ansible/tree/devel/library/module_utils/network/f5